src/content/app/entity-viewer/variant-view/transcript-consequences/TranscriptConsequences.tsx

```diff
@@ -23,13 +23,16 @@ import { getReverseComplement } from 'src/shared/helpers/sequenceHelpers';
 import { pluralise } from 'src/shared/helpers/formatters/pluralisationFormatter';
 import { defaultSort as defaultSortForTranscripts } from 'src/content/app/entity-viewer/shared/helpers/transcripts-sorter';
 
-import { getExpandedTranscriptConseqeuenceIds } from 'src/content/app/entity-viewer/state/variant-view/general/variantViewGeneralSelectors';
-import { setExpandedTranscriptConsequenceIds } from 'src/content/app/entity-viewer/state/variant-view/general/variantViewGeneralSlice';
+import { getExpandedTranscriptConseqeuenceIds } from 'src/content/app/entity-viewer/state/variant-view/transcriptConsequenceSelectors';
+import { toggleTranscriptConsequenceIds } from 'src/content/app/entity-viewer/state/variant-view/transcriptConsequenceSlice';
 import { formatAlleleSequence } from '../variant-view-sidebar/overview/MainAccordion';
 
 import useTranscriptConsequencesData, {
   type TranscriptConsequencesData
 } from './useTranscriptConsequencesData';
+import useExpandedTranscriptConsequence from '../hooks/useExpandedTranscriptConsequence';
+
+import { type GeneInResponse } from '../../state/api/queries/variantTranscriptConsequencesQueries';
```

Comment:
Style.

- If we are only importing a type, and no values, from a module, then prefer `import type { SomeType }` over `import {type SomeType}`.
- Could you move this import down, below components but above css? This is where we usually place type imports.